lib/ansible/playbook/loop_control.py

```diff
@@ -39,3 +39,7 @@ def __init__(self):
     def load(data, variable_manager=None, loader=None):
         t = LoopControl()
         return t.load_data(data, variable_manager=variable_manager, loader=loader)
+
+    def _post_validate_label(self, attr, value, templar):
```

Comment:
Instead of this method, I think we could just set `static=True` on the FA.

Reply:
`static=True` felt confusing as it is allowed for `label` to be a template. Not adding `always_post_validate=True` seems to be have the same effect, so I did that.